src/Artsy/Analytics/Schema/Values.ts

```diff
@@ -36,263 +36,134 @@ export enum OwnerType {
  *       use Click on Force or do we use Tap when browsing from a mobile device?
  */
 export enum ActionType {
-  /**
-   * A click on a UI element using a mouse-like input device.
-   *
-   * TODO: Check if ‘Tap’ and this can be combined.
-   */
-  Click = "Click",
-
-  /**
-   * A click on 'Buy Now' or 'Make offer' buttons.
-   */
+  AuthImpression = "Auth impression",
+  Click = "Click", // TODO: Check if ‘Tap’ and this can be combined.
+  ClickedBid = 'Clicked "Bid"',
   ClickedBuyNow = "Clicked buy now",
-  ClickedMakeOffer = "Clicked make offer",
   ClickedContactGallery = 'Clicked "Contact Gallery"',
-  ClickedBid = 'Clicked "Bid"',
-
-  /**
-   * Triggers a pageview in force, skips segment
-   */
+  ClickedMakeOffer = "Clicked make offer",
   ClickedReadMore = "Clicked read more",
-
   CommercialFilterParamsChanged = "Commercial filter params changed",
-
-  /**
-   * A/B Test Experiments
-   */
   ExperimentViewed = "Experiment Viewed",
-
-  /**
-   * Moving the mouse pointer over a UI element or, when browsing on a mobile
-   * device, by first tapping the UI element once making it switch into
-   * continuous hover mode.
-   */
```

Comment:
I'm not one to delete comments, but these aren't useful within the context of how we work with data-team implementing analytics. They pass us a matrix, we drop the values in. What is useful however is alphabetization, so that we can quickly add a new field or see if something already exists. 

I preserved `TODO`'s as those seem important in terms of future schema work.

Reply:
I think this is more a result of things haven gotten messy and there haven’t been time yet for the data team and us to sit down and tighten the schema. The ones with comments look right, some of the newer ones don’t (e.g. `SubmittedOffer`, which was probably a click with the action `submitted_offer`).

Please revert this for the time being.

Reply:
Done

Reply:
Thanks!